src/index.js

```diff
@@ -3,8 +3,30 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { IntlProvider } from 'react-intl';
+import messages_es from './i18n/es.json';
+import messages_en from './i18n/en.json';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const messages = {
+  es: messages_es,
+  en: messages_en,
+};
+
+//TODO: this hardcoded data will depend by the app language
+const language = (
+  (navigator.languages && navigator.languages[0]) ||
```

Comment:
I don't think you need all this options, I've check here https://www.w3schools.com/jsref/prop_nav_language.asp and with `navigator.language` and `navigator.browserLanguage` we are covered.